include/picongpu/plugins/openPMD/writer/ParticleAttribute.hpp

```diff
@@ -48,6 +48,8 @@ namespace picongpu
         struct ParticleAttribute
         {
             /** write attribute to openPMD series
+             *
+             * @attention This is a MPI_Collective operation and all MPI ranks must participate.
```

Comment:
The only collective part is the creation of the record which I will factor out in a follow-up PR. Fine to leave this comment in for now, but it will be very temporary only.